content/en/docs/reference/glossary/qos-class.md

```diff
@@ -0,0 +1,24 @@
+---
+title: QoSClass
+id: qosClass
```

Comment:
I _think_ we typically use all-lowercase ids for glossary entries.

Reply:
oops, fixing